tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -656,6 +647,13 @@ def calibrate(self,
     # OPs supported by TensorRT but don't have a GPU implementation are allowed
     # to execute on CPU.
     calibrate_config = config_pb2.ConfigProto(allow_soft_placement=True)
+
+    if self._test_only_disable_non_trt_optimizers:
+        calibrate_config = \
+            trt_utils.disable_non_trt_optimizers_in_rewriter_config(
+                calibrate_config
```

Comment:
This doesn't actually work, calibrate_config is NOT a rewriter_config.

Reply:
Oooops. I just pushed a fix